koku/api/user_access/test/test_view.py

```diff
@@ -22,607 +59,255 @@ def setUp(self):
         super().setUp()
         self.client = APIClient()
 
-    @RbacPermissions(
-        {
-            "aws.account": {"read": ["*"]},
-            "aws.organizational_unit": {"read": []},
-            "gcp.account": {"read": []},
-            "gcp.project": {"read": []},
-            "ibm.account": {"read": []},
-            "azure.subscription_guid": {"read": []},
-            "oci.payer_tenant_id": {"read": []},
-            "openshift.cluster": {"read": []},
-            "openshift.node": {"read": []},
-            "openshift.project": {"read": []},
-            "cost_model": {"read": [], "write": []},
-        }
-    )
+    @RbacPermissions(build_rbac_permissions({"aws.account": {"read": ["*"]}}))
     def test_aws_view_read(self):
         """Test user-access view with aws read wildcard permission."""
         url = reverse("user-access")
         response = self.client.get(url, **self.headers)
+        testing_matrix = {"any": {"access": True}, "aws": {"access": True}}
+        expected_output = build_expected_ouput(testing_matrix)
+        for result in response.data.get("data"):
```

Comment:
This could be `None` resulting in a `TypeError`. It might be better to do a direct key lookup so that the test will fail with a `KeyError` is `data` is missing.